README.md

```diff
@@ -1,24 +1,66 @@
-# remem
+<h1 align="center">remem</h1>
+<div align="center">
+ <strong>
+   Thread-safe object reuse
+ </strong>
+</div>
 
-Utility for reusing pieces of memory
+<br />
 
-## Usage
+<div align="center">
+  <!-- Crates version -->
+  <a href="https://crates.io/crates/remem">
+    <img src="https://img.shields.io/crates/v/remem.svg?style=flat-square"
+    alt="Crates.io version" />
+  </a>
+  <!-- Downloads -->
+  <a href="https://crates.io/crates/remem">
+    <img src="https://img.shields.io/crates/d/remem.svg?style=flat-square"
+      alt="Download" />
+  </a>
+  <!-- docs.rs docs -->
+  <a href="https://docs.rs/remem">
+    <img src="https://img.shields.io/badge/docs-latest-blue.svg?style=flat-square"
+      alt="docs.rs docs" />
+  </a>
+</div>
 
-``` rust
-let pool = Pool::<Vec<u8>>::new(|| Vec::new());
-let mut item = pool.get();
+<div align="center">
+  <h3>
+    <a href="https://docs.rs/remem">
+      API Docs
+    </a>
+    <span> | </span>
+    <a href="https://github.com/rylev/remem/blob/master/.github/CONTRIBUTING.md">
+      Contributing
+    </a>
+  </h3>
+</div>
 
-item.push(1);
-item.push(2);
-item.push(3);
+## Installation
 
-drop(item);
+With [cargo add](https://github.com/killercup/cargo-edit) installed run:
 
-// item's memory can now be reused
+```sh
+$ cargo add remem
+```
 
-let mut item = pool.get();
+## Safety
+This crate uses two instances of `unsafe` to implement a lock-free Treiber
+stack. This implementation was copied from the reference implementation in
+`crossbeam`.
 
-item.push(1);
-item.push(2);
-item.push(3);
-```
+## License
+
+<sup>
+Licensed under either of <a href="LICENSE-APACHE">Apache License, Version
```

Comment:
I believe we'll need to add the actual license text to the repo?

Reply:
oh yeah you're right; oops!